Gem/Code/Source/Components/WasdPlayerMovementComponent.cpp

```diff
@@ -70,20 +70,24 @@ namespace MultiplayerSample
         // inputs for your own component always exist
         WasdPlayerMovementComponentNetworkInput* wasdInput = input.FindComponentInput<WasdPlayerMovementComponentNetworkInput>();
 
-        wasdInput->m_forwardAxis = StickAxis(m_forwardWeight - m_backwardWeight);
-        wasdInput->m_strafeAxis = StickAxis(m_leftWeight - m_rightWeight);
+        if (wasdInput)
```

Comment:
Is this really null? Seems like something we should be asserting on.

Reply:
Fair point, this is likely hiding a component init order issue or something.